crates/task/src/debug_format.rs

```diff
@@ -16,13 +16,26 @@ impl Default for DebugConnectionType {
 #[derive(Default, Deserialize, Serialize, PartialEq, Eq, JsonSchema, Clone, Debug)]
 pub struct TCPHost {
     /// The port that the debug adapter is listening on
+    ///
+    /// Default: We will try to find an open port
     pub port: Option<u16>,
     /// The host that the debug adapter is listening too
+    ///
+    /// Default: 127.0.0.1
     pub host: Option<Ipv4Addr>,
-    /// The max amount of time to connect to a tcp DAP before returning an error
+    /// The max amount of time in milliseconds to connect to a tcp DAP before returning an error
+    ///
+    /// Default: 2000ms
     pub timeout: Option<u64>,
```

Comment:
We should probably make timeout a user setting too

Reply:
Should that override evrything or should that be the default/fallback value that is now 2000ms?